modules/renter/hostdb/negotiate_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/NebulousLabs/Sia/modules/gateway"
 	"github.com/NebulousLabs/Sia/modules/miner"
 	"github.com/NebulousLabs/Sia/modules/transactionpool"
-	"github.com/NebulousLabs/Sia/modules/wallet"
+	modWallet "github.com/NebulousLabs/Sia/modules/wallet" // name conflicts with type
```

Comment:
should it be the import that gets renamed or the object that gets renamed? I think I would prefer that the import not be changed, because it's referred to as `wallet` everywhere else in the codebase.